pdf/pdflinkarea.cpp

```diff
@@ -33,7 +34,10 @@ class PDFLinkArea::Private
 
     bool clickInProgress;
     QPointF clickLocation;
+    ulong clickTimestamp;
     int wiggleFactor;
+
+    QTimer *pressTimer;
```

Comment:
I agree, would have prefered it like that also myself, but I copied it from pdfcanvas.cpp. Will change for member variable.